src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Services/ContentApiClient.cs

```diff
@@ -59,4 +59,34 @@ public ContentApiClient(ILogger<ContentApiClient> logger, HttpClient httpClient)
         return publications
             ?? throw new Exception("Could deserialize publications from content API.");
     }
+
+    public async Task<Either<ActionResult, PublicationCacheViewModel>> GetPublication(Guid publicationId)
+    {
+        var response = await _httpClient.GetAsync($"api/publications/{publicationId}");
+
+        if (!response.IsSuccessStatusCode)
+        {
+            var message = await response.Content.ReadAsStringAsync();
+            _logger.LogError(StringExtensions.TrimIndent(
+                $"""
+                 Failed to retrieve publication '{publicationId}' with status code: {response.StatusCode}. Message:
+                 {message}
+                 """));
+
+            switch (response.StatusCode)
+            {
+                case HttpStatusCode.BadRequest:
+                    return new BadRequestObjectResult(await response.Content.ReadFromJsonAsync<ValidationProblemDetails>());
+                case HttpStatusCode.NotFound:
+                    return new NotFoundObjectResult(await response.Content.ReadFromJsonAsync<ProblemDetails>());
+                default:
+                    response.EnsureSuccessStatusCode();
+                    break;
+            }
+        }
+
+        var deserializedResponse = await response.Content.ReadFromJsonAsync<PublicationCacheViewModel>();
+
+        return new Either<ActionResult, PublicationCacheViewModel>(deserializedResponse);
```

Comment:
Similar to `ListPublications`, we have an implicit conversion on Either tasks so we can just do:

```cs
var publication = await response.Content.ReadFromJsonAsync<PublicationCacheViewModel>();

return publication
    ?? throw new Exception("Could deserialize publication from content API.");
```